_pages/home/schematics.md

```diff
@@ -0,0 +1,71 @@
+# Getting Started
+
+This page is intended for customers. If you are looking for a developers guide on how to work with schematics, visit this [page](https://github.com/SAP/spartacus/blob/develop/projects/schematics/README.md).
```

Comment:
As we moved docs here, I believe this link should be removed?

Reply:
The idea is to have two docs:
- one for customers, which will live in this repo
- and one for developers, which will live in our main repo

And the idea is to link them, so that people can easily switch between the two, should they need to.